api/src/main/java/org/hawkular/alerts/actions/api/ActionResponseMessage.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Red Hat, Inc. and/or its affiliates
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
```

Comment:
I think years should be adopted?

Reply:
I think these were autogenerated previously by the license-checker. But I don't know if we're going to keep using it?